deployment/chainloop/Chart.yaml

```diff
@@ -4,7 +4,7 @@ description: Chainloop is an open source software supply chain control plane, a
 
 type: application
 # Bump the patch (not minor, not major) version on each change in the Chart Source code
-version: 1.7.0
+version: 1.7.1
```

Comment:
We'll bundle this change with the automatic PR that will be created when we release next week.